src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -901,6 +907,8 @@
         text: az aks update -g MyResourceGroup -n MyManagedCluster --enable-windows-gmsa
       - name: Enable Windows gmsa for a kubernetes cluster without setting DNS server in the vnet used by the cluster.
         text: az aks update -g MyResourceGroup -n MyManagedCluster --enable-windows-gmsa --gmsa-dns-server "10.240.0.4" --gmsa-root-domain-name "contoso.com"
+      - name: Update a existing managed cluster to a managed cluster snapshot.
+        text: az aks update --cluster-snapshot-id <clustersnapshot resource id> -g MyResourceGroup -n MyManagedCluster
```

Comment:
Maybe add another example under `az aks upgrade`? And from my understanding, the only difference between update and upgrade is that only upgrade allows a different version than the current cluster, and it's just a limitation imposed on the CLI side?

Reply:
other than cluster version, upgrade without --control-plane-only will also upgrade nodepool version.